python/GafferUI/VectorDataWidget.py

```diff
@@ -418,32 +423,41 @@ def editSignal( self ) :
 
 		return self.__editSignal
 
+	## A signal emitted when the user right clicks selected cells.
+	# Slots should accept ( VectorDataWidget, IECore.MenuDefinition ) arguments,
+	# modify the given menu and return it.
+	def dataMenuSignal( self ) :
+
+		return self.__dataMenuSignal
+
 	## Returns a definition for the popup menu - this is called each time
 	# the menu is displayed to allow menus to be built dynamically. May be
 	# overridden in derived classes to modify the menu.
 	## \todo We should remove this as part of implementing #217, and just
 	# let everything hook onto contextMenuSignal() instead.
-	def _contextMenuDefinition( self, selectedRows ) :
+	def _contextMenuDefinition( self, selectedRows, menuDefinition = None ) :
```

Comment:
I don't think there's any need to modify this function, and the `menuDefinition` argument seems more like an implementation detail - not something we want to expose in the API.

Reply:
Fixed and squashed into 71a0621e4ce0494c03be56e1315fa5a62a9ff1f2